gradle/libs.versions.toml

```diff
@@ -34,6 +34,7 @@ hive2 = "2.3.9"
 hadoop2 = "2.10.2"
 hadoop3 = "3.3.0"
```

Comment:
Why do we use "3.3.0", not "3.3.6" as we talked offline? Typically we should not pick ".0" version as it is not stable enough.

Reply:
Why do we use "3.3.0", not "3.3.6" as we talked offline?

Let me check if any severe bugs have been fixed in minor version 3.3.6 and then decide to upgrade to 3.3.6 wholely. I initially only considered 3.3.0 to have all the features we needed and ignore the potential bugs.

Reply:
Please use a separate PR to handle this.